src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -454,6 +454,31 @@ def closeBlockContainer(self, name, scope='cms'):
             self.logger.error("Exception closing container/block: %s. Error: %s", name, str(ex))
         return response
 
+    def moveReplicationRule(self, ruleId, rseExpression, issuer):
+        """
+        Perform move operation for provided rule id and rse expression
+        :param ruleId: rule id
+        :param rseExpression: rse expression
+        :param issuer: rucio quota account
+        :param kwargs: positional arguments for rucio API
+        :return: it returns either an empty list or a list with a string id for the rule created
+        Please note, we made return type from this wrapper compatible with createReplicateRule
+        """
+        ruleIds = []
+        try:
+            rid = self.cli.move_replication_rule(ruleId, rseExpression, issuer)
```

Comment:
Does this API accept move in bulk (many rule ids to be moved in a single HTTP call)? If so, I would suggest to allow ruleId to be a list (so, rename it to ruleIds or rulesList, or else). 
You might want to add a pointer to the Rucio API documentation in the last line of the docstring as well.

Reply:
the rucio move_replication_rule does not deal with bulk operations